core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,70 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A plugin that can efficiently export data from Oracle to be used in Hydrator pipelines.
+Oracle includes command line tools to export data that can be utilized to perform this task.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto hdfs or local file system using an action plugin
+that does not require a JDBC connection to perform the export from Oracle.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote DB machine.
+
+**oracleServerPort:** Port of the remote DB machine.Defaults to 22.
+
+**oracleServerUsername:** Username for remote DB host.
+
+**oracleServerPassword:** Password for remote DB host.
+
+**dbUsername:** Username to connect to oracle DB.
+
+**dbPassword:** Password to connect to oracle DB.
+
+**oracleHome:** Path of the ORACLE_HOME.
+
+**oracleSID:** Oracle SID.
+
+**queryToExecute:** Query to be executed to export.Query should have ';' at the end.
+
+**pathToWriteFinalOutput:** Path where output file to be exported.
+
+**format:** Format of the output file.
```

Comment:
are there known strings for this format? If so, please provide acceptable values/examples in the description.